UCLALibrary/library-website-nuxt
# Set craft token in header for craft preview

This PR sets the craft token in headers for unsaved previews in Craft CMS.
To test this locally 

1. pull this branch
2. change the Craft URL to point to local nitro craft 
3. npm run dev
4. check if craft local nitro site is up nitro ls
5.  change the preview_base_url env to Nuxt app endpoint.
6. Verify if craft side-by-side previews work.
